https://redhat.atlassian.net/browse/ACM-28530 Tshirt obs (#8703)
open-cluster-management/rhacm-docs: observability/obs_config_instance.adoc

:_mod-docs-content-type: PROCEDURE

[#change-obs-instance-size]
= Changing observability instance sizes

[role="_abstract"]

> The observability instance sizes feature simplifies the scaling process for {acm} observability. With T-shirt sizing, you can scale all components to preconfigured settings instead of scaling each component individually.

Reviewer: ```suggestion
The observability instance sizes feature simplifies the scaling process for {acm} observability. With T-shirt sizing, you can scale Observability API and Thanos components to preconfigured settings instead of scaling each component individually.
```

The instance size feature is optional.

*Required access:* Cluster administrator

.Prerequisites

- Your hub cluster needs sufficient CPU and memory resources for the instance size that you want to use. See xref:../observability/obs_instance_sizes.adoc#obs-instance-sizes[Observability instance sizes] for more information.
- You must correctly configure persistent volumes for stateful components such as `alertmanager`, `thanos-compactor`, and `thanos-receive`.

.Procedure

To apply a size, edit the `MultiClusterObservability` custom resource on your hub cluster. Complete the following step:

. Add the instance size that you want to use to the `spec.instanceSize` field in the `MultiClusterObservability` custom resource. See the following YAML sample:

+
[source,yaml]
----
apiVersion: observability.open-cluster-management.io/v1beta2
kind: MultiClusterObservability
metadata:
  name: observability
spec:
  instanceSize: <size_value>
----

+
* For a list of valid `spec.instanceSize` options, see xref:../observability/obs_instance_sizes.adoc#obs-instance-sizes[Observability instance sizes].

.Verification

> After applying a new size, you can verify the status and instance size of the observability pods in the console.

Reviewer: ```suggestion
After applying a new size, you can verify the status and instance size of the observability pods in the `open-cluster-management-observability` namespace in the console.
```

:_mod-docs-content-type: PROCEDURE

[#change-obs-instance-size]
= Changing observability instance sizes

[role="_abstract"]

> The observability instance sizes feature simplifies the scaling process for {acm} observability. With T-shirt sizing, you can scale Observability API and Thanos components to preconfigured settings instead of scaling each component individually.

Reviewer 2: It seems odd to have t-shirt mentioned. Let's stay consistent and say "instance sizing". This is literally the only time "T-shirt" is mentioned in this file

Author: That makes, I'll remove it!

> The observability instance sizes feature simplifies the scaling process for {acm} observability. With instance sizes, you can scale Observability API and Thanos components to preconfigured settings instead of scaling each component individually.

Reviewer 2: ```suggestion
The observability instance sizes feature simplifies the scaling process for {acm} observability. With instance sizes, you can scale observability API and Thanos components to preconfigured settings instead of scaling each component individually.
```

The instance size feature is optional.

*Required access:* Cluster administrator

.Prerequisites

> - Your hub cluster needs sufficient CPU and memory resources for the instance size that you want to use. See xref:../observability/obs_instance_sizes.adoc#obs-instance-sizes[Observability instance sizes] for more information.

Reviewer 2: ```suggestion
- Your hub cluster needs sufficient CPU and memory resources for the instance size that you want to use. See xref:../observability/obs_instance_size.adoc#obs-instance-sizes[Observability instance sizes] for more information.
```

- You must correctly configure persistent volumes for stateful components such as `alertmanager`, `thanos-compactor`, and `thanos-receive`.

.Procedure

To apply a size, edit the `MultiClusterObservability` custom resource on your hub cluster. Complete the following step:

. Add the instance size that you want to use to the `spec.instanceSize` field in the `MultiClusterObservability` custom resource. See the following YAML sample:

+
[source,yaml]
----
apiVersion: observability.open-cluster-management.io/v1beta2
kind: MultiClusterObservability
metadata:
  name: observability
spec:
  instanceSize: <size_value>
----

+

> * For a list of valid `spec.instanceSize` options, see xref:../observability/obs_instance_sizes.adoc#obs-instance-sizes[Observability instance sizes].

Reviewer 2: ```suggestion
* For a list of valid `spec.instanceSize` options, see xref:../observability/obs_instance_size.adoc#obs-instance-sizes[Observability instance sizes].
```

.Verification

After applying a new size, you can verify the status and instance size of the observability pods in the `open-cluster-management-observability` namespace in the console.

*Important Notes:*

* If you manually defined replicas or resource limits in the `AdvancedConfig` section of you `MultiClusterObservability` custom resource, your existing `AdvancedConfig` settings always override the `instanceSize` setting.

> * The default size aligns with the standard {acm-short} installation requirements of 2701mCPU and 11972Mi memory for a five cluster environment. Due to variations in cluster size, number of nodes, load, and additional factors, only an approximation of the number of timeseries are provided.

Reviewer 2: Broken links

<img width="818" height="660" alt="Image" src="https://github.com/user-attachments/assets/53ceb8a6-53e7-4419-8e55-c3615006d8c9" />

Author: I think it's fixed now
